a11y.js

```diff
@@ -77,13 +78,19 @@ const pageTypes = {
   idxPage: ["//div[@id='root']/header/nav/div/div[1]/div/ul"],
   onDemandTV: ['/iframe'],
   mediaAssetPage: [...mediaEmbedErrorsToSuppress, '/iframe'],
+  featureIndexPage: [],
 };
 
-Object.keys(pageTypes).forEach(pageType => {
```

Comment:
Please can you help me understand why this block has been refactored? Has the value returned by `getPageUrls` changed?

Reply:
I noticed that the tests weren't running properly if there were services / page types with multiple URLs - it was testing against a 404 page. This change was to ensure that each path returned by getPageUrls was tested individually.

Reply:
Ah, thanks for explaining, Karina.